src/integration/pages/storyPage/canonicalTests.js

```diff
@@ -38,4 +38,17 @@ export default () => {
       });
     }
   });
+
+  describe('Ads', () => {
+    const hasAds = service === 'mundo';
+    if (hasAds) {
+      const leaderboardEl = document.getElementById('dotcom-leaderboard');
+      const mpuEl = document.getElementById('dotcom-mpu');
+
+      it('should have ads in the document', () => {
```

Comment:
Do we have a test that asserts our production toggle is working as expected?

Reply:
@simonsinclair when you say "production toggle" do you mean the `process.env` value or the iSite one ?

Reply:
Yes, `process.env.NODE_ENV`. Similar to Andrew's suggestion below - we should probably assert that this control is working, so we do not render STY Ads on live prior to release.

Reply:
@simonsinclair done.